src/main/java/echobot/Command.java

```diff
@@ -0,0 +1,367 @@
+            System.out.println("____________________________________________________________");
+            System.out.println(" The description of an event must include start and end times.");
+            System.out.println("____________________________________________________________");
+        } else {
+            String[] details = inputParts[1].split(" /from | /to ");
+            if (details.length < 3 || details[0].trim().isEmpty()) {
+                System.out.println("____________________________________________________________");
+                System.out.println(" The description of an event cannot be empty.");
+                System.out.println("____________________________________________________________");
+            } else {
+                tasks.addTask(new Event(details[0], details[1], details[2]));
+                System.out.println("____________________________________________________________");
+                System.out.println(" Got it. I've added this task:");
+                System.out.println("   " + tasks.get(tasks.size() - 1));
+                System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+                System.out.println("____________________________________________________________");
+            }
+        }
+    }
+
+    /**
+     * Handles the command to delete a task.
+     *
+     * @param inputParts The parts of the user input, including the task number to be deleted.
+     */
+    private void handleDeleteCommand(String[] inputParts) {
+        if (inputParts.length < 2) {
+            System.out.println("____________________________________________________________");
+            System.out.println(" Please specify the task number to delete.");
+            System.out.println("____________________________________________________________");
+        } else {
+            try {
+                int taskNumber = Integer.parseInt(inputParts[1]) - 1;
+                if (taskNumber < tasks.size() && taskNumber >= 0) {
+                    Task removedTask = tasks.removeTask(taskNumber);
+                    System.out.println("____________________________________________________________");
+                    System.out.println(" Noted. I've removed this task:");
+                    System.out.println("   " + removedTask);
+                    System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+                    System.out.println("____________________________________________________________");
+                } else {
+                    System.out.println("____________________________________________________________");
+                    System.out.println(" Oops! That task number doesn't exist.");
+                    System.out.println("____________________________________________________________");
+                }
+            }
+            catch (NumberFormatException e) {
```

Comment:
Indent your catch line to be on the same line as the closing braces of the try block.